components/popover_list_members/popover_list_members.tsx

```diff
@@ -43,70 +42,53 @@ export type Props = {
     teamUrl?: string;
 }
 
-export type State = {
-    showPopover: boolean;
-    users: UserProfile[];
-    statuses: RelationOneToOne<UserProfile, string>;
-}
-
-export default class PopoverListMembers extends React.PureComponent<Props, State> {
-    private membersList: React.RefObject<HTMLDivElement>;
+export default function PopoverListMembers(props: Props) {
+    const membersListRef: React.RefObject<HTMLDivElement> = React.createRef();
+    const targetRef: React.RefObject<HTMLButtonElement> = React.createRef();
 
-    constructor(props: Props) {
-        super(props);
-        this.membersList = React.createRef();
+    const [showPopover, setShowPopover] = useState(false);
+    const [users, setUsers] = useState<UserProfile[]>(props.users);
+    const [statuses, setStatuses] = useState<RelationOneToOne<UserProfile, string>>(props.statuses);
 
-        this.state = {
-            showPopover: false,
-            users: props.users,
-            statuses: props.statuses,
-        };
-    }
+    useEffect(() => {
+        setUsers(props.users);
+        setStatuses(props.statuses);
+    }, [props.users, props.statuses]);
```

Comment:
I suppose this is the correct way to convert `getDerivedStateFromProps`. 

However, I have a doubt if we still need this because I don't see any where `users` or `statuses` states are being used. Please let me know if I am missing sth.

This is the reason of lint failing:
- `  50:12  error  'users' is assigned a value but never used     @typescript-eslint/no-unused-vars`
- `  51:12  error  'statuses' is assigned a value but never used  @typescript-eslint/no-unused-vars`

Reply:
I think you could actually get rid of both of those from state. I don't know why the old version used `getDerivedStateFromProps` since it doesn't look like it used these either. If I had to guess, it probably predates Redux and just never got cleaned up.

Reply:
that's what I thought as well ... might be some leftover that never got cleaned up

Reply:
Thanks to both of you. 
- `users` is deleted both from `props` and `state`. I also updated index file to remove `users` as we dont need anymore
- `statuses` is deleted from `state` but kept in props